src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-tutorials/opentelemetry-tutorial-java.mdx

```diff
@@ -0,0 +1,1132 @@
+  src={opentelemetryExceptiononSpanEvent}
+/>
+
+To view additional details that you set, such as the span attribute, span name, and status code, click on the **Attributes** tab. This pane is also where you can view additional metadata that is automatically collected by the instrumentation libraries you used in this guide, as well as metadata that is attached by New Relic:
+
+<img
+  title="Screenshot of the attributes tab"
+  alt="Screenshot of the attributes tab"
+  src={opentelemetryAttributesTab}
+/>
+
+For more details about viewing your data, see [OpenTelemetry in the New Relic UI](https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-distributed-tracing-page)
+
+
+### Metrics [#metrics]
+
+Once you've reached the `getting-started-java` entity in New Relic, you can see a list of all collected metrics, such as runtime metrics (JVMs) and your custom counter attributes.
+
+#### Metrics explorer [#metrics-explorer]
+
+This is a tool that allows you to see a list of your metrics.
+
+1. In the left pane, select **Data > Metrics explorer**, and then select `fibonacci.invocations`.
+
+    <img
+      title="Screenshot showing metrics explorer"
+      alt="Screenshot showing metrics explorer"
+      src={opentelemetryMetricsExplorerMain}
+    />
+2. Under **Dimensions**, view the attributes you collected along with your custom metric, and then click on **fibonacci.valid.n**.
+
+    <img
+      title="Screenshot showing the total number of invocations of fibonacci function"
+      alt="Screenshot showing the total number of invocations of fibonacci function"
+      src={opentelemetryMetricsExplorerDimensions}
+    />
+
+Learn more about the metrics explorer view [here](docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-metrics-explorer-page).
+
+
+#### JVMs [#jvms]
+
+You can also view visualizations of your Java runtime metrics, including throughput, memory usage, and garbage collection time per minute, by heading to the JVM page:
+
+<img
+  title="Screenshot of the main JVMs page"
+  alt="Screenshot of the main JVMs page"
+  src={opentelemetryJvmsPageMain}
+/>
+
+You can view the metrics for a single instance, or select multiple instances to compare them:
+
+<img
+  title="Screenshot of how to compare JVM instances"
+  alt="Screenshot of how to compare JVM instances"
+  src={opentelemetryCompareJvmInstances}
+/>
+
+Here is what you will see after selecting your instances and clicking **Compare**, where each instance is color coded for easy identification:
+
+<img
+  title="Screenshot showing the color-coded JVM comparison"
+  alt="Screenshot showing the color-coded JVM comparison"
+  src={opentelemetryJvmColorCodedComparison}
+/>
+
+To learn more about the JVMs view, see [here](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-jvms-page/).
```

Comment:
reminder about descriptive url text (i'll stop mentioning that now)

Reply:
@zuluecho9 thanks for the reminder!